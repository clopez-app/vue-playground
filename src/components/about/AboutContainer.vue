<template>
  <h1>About</h1>
  <div class="intro">
    <p>
      This playground was made to refresh and learn the configuration and usage
      of Vue in a proyect.
    </p>
  </div>
</template>

<script setup lang="ts"></script>
