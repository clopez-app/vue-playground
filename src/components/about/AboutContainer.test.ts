import AboutContainer from './AboutContainer.vue';
import { mount } from '@vue/test-utils';

describe('About component', () => {
  test('renders correctly', () => {
    expect(AboutContainer).toBeTruthy();

    const wrapper = mount(AboutContainer);

    expect(wrapper.find('h1').text()).toBe('About');
    expect(wrapper.find('p').text()).toContain('playground');
    expect(wrapper.html()).toMatchSnapshot();
  });
});
