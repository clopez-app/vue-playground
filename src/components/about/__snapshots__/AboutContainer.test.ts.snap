// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About component > renders correctly 1`] = `
"<h1>About</h1>
<div class=\\"intro\\">
  <p> This playground was made to refresh and learn the configuration and usage of Vue in a proyect. </p>
</div>"
`;
